react/src/lib/components/SubsurfaceViewer/layers/map/mapLayer.ts

```diff
@@ -40,7 +40,7 @@ export type Params = {
 
 async function load_mesh_and_properties(
     meshUrl: string,
-    propertiesUrl: string,
+    propertiesUrl: string | number[],
```

Comment:
If properties now can be not only URL, is the name "propertiesUrl" still appropriate?

Also, is it possible to use the Float32Array directly in order to enforce 32 bit float from the client directly, which could avoid confusion caused by truncation, say if double precision was used?